pkg/apis/servicecatalog/v1alpha1/doc.go

```diff
@@ -19,5 +19,7 @@ limitations under the License.
 // +k8s:openapi-gen=true
 // +k8s:defaulter-gen=TypeMeta
 
+// Package v1alpha1 defines all of the versioned (v1alpha1) definitions
+// of the service catalog model.
```

Comment:
We found some extreme brittleness in this file around adding doc comments - do regenerate the code and ensure that nothing has changed.

Also, we need a 'verify' script for the generated code.